variables.tf

```diff
@@ -17,3 +17,8 @@ variable "image" {
 variable "instance_size" {
   default = "BUILD_GENERAL1_SMALL"
 }
+
+variable "buildspec" {
+  default     = ""
+  description = "The build spec declaration to use for this build project's related builds"
```

Comment:
According to @robinbowes we can define `buildspec=""` and it defaults to the version in the source code. See: https://github.com/cloudposse/tf_codebuild/pull/2#issuecomment-322435270